CRITICAL - 2022-06-30 15:48:07 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:08 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:09 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:10 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:10 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:11 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:12 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:13 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:13 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 15:48:14 --> mysqli_connect(): (HY000/1045): Access denied for user '......'@'localhost' (using password: YES)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', '/home/fallenxt/...', 8, Array)
#1 /home/fallenxt/public_html/app/Controllers/conn.php(8): mysqli_connect('localhost', '......', '.....', '.....')
#2 /home/fallenxt/public_html/app/Controllers/Connect.php(13): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(801): App\Controllers\Connect->__construct()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(401): CodeIgniter\CodeIgniter->createController()
#5 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-06-30 17:11:33 --> Access denied for user 'fallenxt_jangrashab'@'localhost' (using password: YES)
#0 /home/fallenxt/public_html/app/Views/Server/conn.php(8): mysqli_connect('localhost', 'fallenxt_jangra...', '0okm)OKM$14', 'fallenxt_jangra...')
#1 /home/fallenxt/public_html/app/Views/Server/Server.php(3): include('/home/fallenxt/...')
#2 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/View/View.php(214): include('/home/fallenxt/...')
#3 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/View/View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/Common.php(1097): CodeIgniter\View\View->render('Server/Server', Array, true)
#5 /home/fallenxt/public_html/app/Controllers/User.php(304): view('Server/Server', Array)
#6 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(824): App\Controllers\User->Server()
#7 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 /home/fallenxt/public_html/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /home/fallenxt/public_html/public/index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-06-30 17:28:44 --> Cannot redeclare App\Controllers\Keys::api_get_keys()
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2022-06-30 17:38:21 --> Cannot redeclare App\Controllers\User::alterUser()
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
